inventory/libvirt/hosts/libvirt_generic.py

```diff
@@ -0,0 +1,179 @@
+#!/usr/bin/env python
```

Comment:
@jwhonce iirc, the issue was around maintenance, since we will be updating the scripts from upstream from time to time.  I see no issues with doing it if it helps some users... Also, the libvirt inventory script only exists as a [PR](https://github.com/ansible/ansible/pull/10686) for the ansible repo currently :)